source/common/tracing/http_tracer_impl.cc

```diff
@@ -66,7 +66,7 @@ Decision HttpTracerUtility::isTracing(const StreamInfo::StreamInfo& stream_info,
 
   // TODO PERF: Avoid copy.
```

Comment:
I think you fixed this?

Reply:
Indeed I did now that `isTraceableUuid` takes `string_view` rather than `std::string`.  Removing the comment, taking the credit (I don't see an existing issue for this, if there is feel free to close it).